doc/_stdlib_gen/stdlib-content.jsonnet

```diff
@@ -722,6 +725,46 @@ local html = import 'html.libsonnet';
                   }
               }
             |||),
+            html.p({}, |||
```

Comment:
Any reason to put examples in the description instead of using `examples` field? The latter is generally better, because it allows us to change the presentation, e.g. we want to make the examples interactive one day. 

See: `manifestTomlEx`.

FYI The only reason `manifestXmlJsonml` and some others keep everything in the description is that they have some elements which are not supported properly yet (e.g. an svg picture illustrating an example).

We should add some documentation about this file, sorry for the burden of figuring it out based on examples alone.

Reply:
@sbarzowski I took a second look, it seems the `examples` field does not preserve newlines. The example for `manifestTomlEx` is flattened a single line and seems hard to read. What are your thoughts?

Reply:
Please leave it in `examples` and I'll fix the formatting later.